docs/node/run-your-node/paratime-node.mdx

```diff
@@ -332,7 +332,7 @@ Before using this configuration you should collect the following information to
 
 :::caution
 
-Make sure the `worker.p2p.port` (default: `9200`) port is exposed and publicly
+Make sure the `p2p.port` (default: `30002`) port is exposed and publicly
```

Comment:
Ah, actually I just checked and 9200 is still the default port (if not set in config), it is just that the above example sets it to 30002. So maybe to make it more clear this should be "defaults to `9200` if not set in config"?

Reply:
Updated the port in the sample config and note to 9200